sdcm/cluster.py

```diff
@@ -710,6 +710,57 @@ def stop_task_threads(self, timeout=10):
             self._journal_thread.join(timeout)
         del self.remoter
 
+    def get_cpumodel(self):
+        """Get cpu model from /proc/cpuinfo
+
+        Get cpu model from /proc/cpuinfo of node
+        """
+        cpuinfo_cmd = 'cat /proc/cpuinfo'
+
+        try:
+            result = self.remoter.run(cmd=cpuinfo_cmd, ignore_status=True, verbose=False)
+            model = re.findall('^model\sname\s:(.+)$', result.stdout.strip(), re.MULTILINE)
+            return model[0].strip()
+
+        except Exception as details:
+            self.log.error('Error retrieving number of core dumps : %s',
+                           details)
+            return None
+
+    def get_installed_packets(self):
+        """Get installed packages on node
+
+        Execute packet manager command and get
+        list of all installed packages
+        """
+        if self.is_ubuntu() or self.is_debian():
+            cmd = 'dpkg-query --show'
+        else:
+            cmd = 'rpm -qa'
+
+        try:
+            result = self.remoter.run(cmd, ignore_status=True, verbose=False)
```

Comment:
if you use `ignore_status=True` it will not throw Exception

Reply:
Think about it again. Remove the ignore flag